napkon_string_matching/terminology/loinc_search.py

```diff
@@ -118,5 +167,16 @@ def start_search_session(search_terms: list = []):
 
 
 if __name__ == "__main__":
+    search_strings = []
     # example call
-    start_search_session(["systolischer Blutdruck", "COVID"])
+    for item in EXAMPLEOUTCOME_ITEMS:
+        # tokenize according to space
+        tokens = item.split(" ")
+        # make tokens alphanumeric
+        tokens = [make_alphanumeric(t) for t in tokens]
+        # remove stop words
+        tokens = remove_stop_words(tokens=tokens)
+        # create search syntax string and append to list
+        search_strings.append(get_search_syntax_string(tokens=tokens))
+    # starting session and search getting one df per search
+    search_results = start_search_session(search_strings)
```

Comment:
Would be nice to have this in a function that is just called by the main context.